src/dupradar/dupradar.r

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env Rscript
+
+## VIASH START
+par <- list(
+  "input" = 'test_data/sample.bam',
+  "id" = 'test',
+  "gtf_annotation" = 'test_data/genes.gtf',
+  "strandedness" = 1,
+  "paired" = TRUE,
+  "threads" = 1,
+  "output_dupmatrix"="dup_matrix.txt",
+  "output_dup_intercept_mqc"="dup_intercept_mqc.txt",
+  "output_duprate_exp_boxplot"="duprate_exp_boxplot.pdf",
+  "output_duprate_exp_densplot"="duprate_exp_densityplot.pdf",
+  "output_duprate_exp_denscurve_mqc"="duprate_exp_density_curve_mqc.pdf",
+  "output_expression_histogram"="expression_hist.pdf",
+  "output_intercept_slope"="intercept_slope.txt"
+)
+## VIASH END
+
+
+if (length(par) < 5) {
+    stop("Usage: dupRadar.r <input.bam> <sample_id> <annotation.gtf> <strandDirection:0=unstranded/1=forward/2=reverse> <paired/single> <nbThreads> <R-package-location (optional)>", call.=FALSE)
+}
+
+
+input_bam <- par$input
+output_prefix <- par$id
+annotation_gtf <- par$gtf_annotation
+stranded <- ifelse(is.null(par$strandedness), 0, as.integer(par$strandedness))
```

Comment:
It may be worthwhile to load `library(rlang)` and use

```suggestion
stranded <- par$strandedness %||% 0L
```

In addition, you don't need to convert the strandedness to an integer using `as.integer` because strandedness is already an integer according to the viash config